src/core/fp.ts

```
@@ -0,0 +1,95 @@
+}
+
+/**
+ * Function composition from right to left
+ */
+export const compose = <A>(...fns: Array<(a: A) => A>): ((a: A) => A) =>
+  fns.reduce((f, g) => x => f(g(x)));
+
+/**
+ * Creates a curried version of a function
+ */
+export const curry = <A extends any[], R>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)